app-js/Controller/LoginController.js

```diff
@@ -1,30 +1,39 @@
+/*
+ * @name Notes.loginController
+*/
 Notes.loginController = {
     loginView: null,
+    view: null,
+    homeView: null,
     
     init: function () {
-	this.loginView = new Notes.LoginView(function (e) {
+        console.log("In Login Controller init function");
+        this.loginView = new Notes.LoginView();
+        this.loginView.show();
+        
+        this.loginView = new Notes.LoginView(
+            function (e, self) {
             console.log(e);
-            Notes.loginController.loginView.resetData();
-        }, function (e) {
+            self.resetData();
+        },
+        function (e, self) {
             console.log(e);
+            
             //read data from View
-            var userModel = Notes.loginController.loginView.readUserData();
+            var userModel = self.readUserData();
+        
             //call api
-
             Notes.utils.post("/api/session", userModel, true, function (response) {
-                Notes.loginController.loginView.hide(response);
-                console.log("OnSuccess Response:", response);
-                //transfer control to notes controller
-                Notes.notesController.init();
-            }, function (response) {
-                console.log("OnFailure Response:", response);
-                Notes.loginController.loginView.showError(response);
-            });
-
+            self.hide(response);
```

Comment:
Are we going to reuse same login view again? If not we should destroy it. So that it will not use memory.